client/src/components/users/TagProfile.vue

```diff
@@ -33,6 +33,17 @@ function translateProfile(profile: UserProfile): string {
   }
   return '';
 }
+
+function getClassForProfile(profile: UserProfile): string {
+  if (profile === UserProfile.Admin) {
+    return 'admin';
+  } else if (profile === UserProfile.Standard) {
+    return 'standard';
+  } else if (profile === UserProfile.Outsider) {
+    return 'outsider';
+  }
+  return '';
```

Comment:
works but this case is clearly a switch case statement, should be replaced by something like:
```
switch (profile) {
  case UserProfile.Admin:
    return 'admin';
  case UserProfile.Standard:
    return 'standard';
  case UserProfile.Outsider:
    return 'outsider';
  default:
    return '';
}
```

Reply:
Done